docs/Guide/preconditions/reporting-precondition-failure.mdx

```diff
@@ -0,0 +1,241 @@
+contain information about the context in which the error was thrown, and the value can be absolutely anything.
+
+We can take advantage of this by adding `context: { silent: true }` to the [`this.error()`][preconditionerror] options.
+We'll use the `OwnerOnly` precondition we made in[Creating Preconditions][creating-preconditions] to demonstrate this.
+
+<Tabs groupId="language-choice">
+<TabItem value="javascript" label="JavaScript" default>
+
+```javascript {9}
+const { Precondition } = require('@sapphire/framework');
+
+module.exports = class OwnerOnlyPrecondition extends Precondition {
+  public run(message) {
+    return message.author.id === 'YOUR_ID'
+      ? this.ok()
+      : this.error({
+          message: 'Only the bot owner can use this command!'
+          context: { silent: true }
+        })
+  }
+};
+```
+
+</TabItem>
+<TabItem value="ESM" label="ESM">
+
+```javascript {9}
+import { Precondition } from '@sapphire/framework';
+
+export class OwnerOnlyPrecondition extends Precondition {
+  public run(message) {
+    return message.author.id === 'YOUR_ID'
+      ? this.ok()
+      : this.error({
+          message: 'Only the bot owner can use this command!',
+          context: { silent: true }
+        })
+  }
+}
+```
+
+</TabItem>
+<TabItem value="typescript" label="TypeScript">
+
+```typescript {10}
+import type { Message } from '@sapphire/framework';
+import { Precondition } from '@sapphire/framework';
+
+export class OwnerOnlyPrecondition extends Precondition {
+  public run(message: Message) {
+    return message.author.id === 'YOUR_ID'
+      ? this.ok()
+      : this.error({
+          message: 'Only the bot owner can use this command!'
+          context: { silent: true }
+        })
+  }
+}
+```
+
+</TabItem>
+</Tabs>
+
+We can then check if this property exists on the error in our listener, and ignore the failure if we find it.
+
+<Tabs groupId="language-choice">
+<TabItem value="javascript" label="JavaScript" default>
+
+```javascript {5}
+const { Listener } = require('@sapphire/framework');
+
+module.exports = class CommandDeniedListener extends Listener {
+  run(error, { message }) {
+    if (error.context.silent) return;
+    return message.channel.send(error.message);
+  }
+};
+```
+
+</TabItem>
+<TabItem value="ESM" label="ESM">
+
+```javascript {5}
+import { Listener } from '@sapphire/framework';
+
+export class CommandDeniedListener extends Listener {
+  run(error, { message }) {
+    if (error.context.silent) return;
+    return message.channel.send(error.message);
+  }
+}
+```
+
+</TabItem>
+<TabItem value="typescript" label="TypeScript">
+
+```typescript {8}
+import type { UserError, CommandDeniedPayload } from '@sapphire/framework';
+import { Listener } from '@sapphire/framework';
+
+export class CommandDeniedListener extends Listener {
+  public run(error: UserError, { message }: CommandDeniedPayload) {
+    // A bit of a hack is required for TypeScript
+    // Since `error.context` is of type `unknown`
```

Comment:
We could always make CommandDeniedPayload take in a generic for the context, which would be better than this hack

Reply:
I'll submit a pr hehe